docs/datagatherers/aks.md

```diff
@@ -0,0 +1,36 @@
+# AKS Data Gatherer
+
+The AKS *data gatherer* fetches information about a cluster from the Azure
+Kubernetes Service API.
+
+## Data
+
+Preflight collects data about clusters. The fields included here can be found
+[here](https://docs.microsoft.com/en-us/rest/api/aks/managedclusters/get).
+
+## Configuration
+
+To use the AKS data gatherer add an `aks` entry to the `data-gatherers`
+configuration. For example:
+
+```
+data-gatherers:
+- kind: "aks"
+  name: "aks"
+  config:
```

Comment:
Something this highlights is that how we configure data gatherers is now different in the CLI and the agent.

Reply:
That's ok. It had to change because ultimately we aim to support multiple instances of every data-gatherer in the agent.